lib/src/arithmetic/values/floating_point_values/floating_point_value.dart

```diff
@@ -14,625 +14,201 @@ import 'package:meta/meta.dart';
 import 'package:rohd/rohd.dart';
 import 'package:rohd_hcl/rohd_hcl.dart';
 
-/// Critical threshold constants
-enum FloatingPointConstants {
-  /// smallest possible number
-  negativeInfinity,
-
-  /// The number zero, negative form
-  negativeZero,
-
-  /// The number zero, positive form
-  positiveZero,
-
-  /// Smallest possible number, most exponent negative, LSB set in mantissa
-  smallestPositiveSubnormal,
-
-  /// Largest possible subnormal, most negative exponent, mantissa all 1s
-  largestPositiveSubnormal,
-
-  /// Smallest possible positive number, most negative exponent, mantissa is 0
-  smallestPositiveNormal,
-
-  /// Largest number smaller than one
-  largestLessThanOne,
-
-  /// The number one
-  one,
```

Comment:
As silly as it seems, we should add zero.

Reply:
Is that different from positive zero?

```
  /// The number zero, negative form.
  negativeZero,

  /// The number zero, positive form.
  positiveZero,
```

Reply:
oh that's right.  I forgot we had added.